roles/tas_single_node/tasks/certificates.yml

```diff
@@ -255,11 +255,63 @@
             (ec_key_cert_match_check.stdout_lines and ec_key_cert_match_check.stdout_lines[0].strip() != ec_key_cert_match_check.stdout_lines[1].strip())
         )
 
-- name: Create CTLog root
+- name: Custom CTLog Root
```

Comment:
I'd suggest using more descriptive names throughout this file; for instance this one is missing a verb; some of the ones below seem to say they create a key, when in fact they create a key *file*. Some of the task names say they "track" something, but I don't think it's really clear from the name what they do.

Reply:
No problem, so in terms of the tracking tasks, those are a list of paths to custom keys. Will run through it now.

Reply:
:heavy_check_mark:

Reply:
I'd still prefer to improve on this. Specific suggestions for this block:
* For the tasks that create the files, add this information specifically to the task name, e.g. `Create Custom CTlog Public Key` => `Create Custom CTlog Public Key file` (also we shouldn't capitalize words "custom", "public" etc)
* `Track CTlog <something>` => `Find all CTlog <something>`